### ISSUE TITLE
 ButtonCommand is disabled when the textbox is read only 

### ISSUE THREAD

**Issue by YounesseSoft** [MahApps/MahApps.Metro#3055]
Hi everyone , I do not want the button to be disabled , when the text read only this is my code please help me .

```
  <TextBox Text="{Binding UrlVideo}"  TextWrapping="Wrap" Margin="9.6,6,9.6,8.8" Style="{DynamicResource MetroButtonTextBox}"
         controls:TextBoxHelper.ButtonCommand="{Binding TextBoxButtonCmd, Mode=OneWay}"
          
         controls:TextBoxHelper.Watermark="مسار الملف الفيديو فارغ" Grid.Row="1" Grid.ColumnSpan="6" FontFamily="El Messiri" FontSize="16" Grid.Column="1" >
            <controls:TextBoxHelper.ButtonContent  >
                <iconPacks:PackIconFontAwesome  Kind="PlayCircleOutline" Height="22" Width="20"/>
            </controls:TextBoxHelper.ButtonContent>
        </TextBox>
```

**Comment by punker76** [comment]
@YounesseSoft This should be already fixed, try using latest v1.5.0. If you set `IsReadOnly="True"` and the `TextBoxHelper.ButtonCommand` is set then it should be clickable.

**Comment by YounesseSoft** [comment]
thank you  brother it helps me really , very nice.........
